src/App.jsx

```diff
@@ -197,7 +188,7 @@ function App() {
     <ThemeProvider theme={themeMode}>
       <CssBaseline />
       {/* {isAppLoading && <LoadingSplash />} */}
-      <div className={`app ${isSmallerScreen && "tablet"} ${isSmallScreen && "mobile"}`}>
+      <div className={`app ${isSmallerScreen && "tablet"} ${isSmallScreen && "mobile"} ${theme}`}>
```

Comment:
check

Reply:
added in https://github.com/OlympusDAO/olympus-frontend/commit/dcf0852d90d53ab3a2e1d98e88f7c648f02c15be